### ISSUE TITLE
Ruby 3.0 and ERB.new issues

### ISSUE THREAD

**Issue by ttscoff** [davetron5000/gli#301]
GLI 2.19.2

```
/usr/local/lib/ruby/gems/3.0.0/gems/gli-2.19.2/lib/gli/commands/help_modules/global_help_format.rb:37: warning: Passing safe_level with the 2nd argument of ERB.new is deprecated. Do not use it, and specify other arguments as keyword arguments.

/usr/local/lib/ruby/gems/3.0.0/gems/gli-2.19.2/lib/gli/commands/help_modules/global_help_format.rb:37: warning: Passing trim_mode with the 3rd argument of ERB.new is deprecated. Use keyword argument like ERB.new(str, trim_mode: ...) instead.

/usr/local/lib/ruby/gems/3.0.0/gems/gli-2.19.2/lib/gli/commands/help_modules/command_help_format.rb:28: warning: Passing safe_level with the 2nd argument of ERB.new is deprecated. Do not use it, and specify other arguments as keyword arguments.
```

Any chance this will be fixed?

**Comment by davetron5000** [comment]
I'll look into it soon. I want GLI to work on as many rubies as I have time to support!

**Comment by ttscoff** [comment]
You rock!

**Comment by davetron5000** [comment]
Welp, this is an easy fix, but I can no longer run the test suite in 3.0 because of some cucumber issue.

Do you have the ability to pull in a branch version of GLI to try to see if it works in your app?

https://github.com/davetron5000/gli/tree/ruby-3

Has the fix for the ERB warning.  It passes unit tests.  I will see about getting the test suite/dev env working but if you can confirm it's working I can do a point release.

**Comment by ttscoff** [comment]
I'm using GLI as a bundled gem. Does gem have an easy option for using a branch as a source? I'll figure it out one way or another.

**Comment by davetron5000** [comment]
You can specify it in your Gemfile and run it locally out of your apps' source code, but I don't think you could build/deploy your app with that.

**Comment by ttscoff** [comment]
Figured out how to do it with gem, seems to be working great!

**Comment by davetron5000** [comment]
OK, this is fixed in 2.20.0
